docs/src/Atmos/Model/risingbubble.md

```diff
@@ -0,0 +1,447 @@
+```@meta
+EditURL = "<unknown>/experiments/AtmosLES/risingbubble.jl"
+```
+
+# Example - Rising Thermal Bubble
+
+In this example, we demonstrate the usage of the CLIMA [AtmosModel](../Atmos/Model/AtmosModel.md)
+machinery to solve for a the fluid dynamics of a thermal perturbation
+in a neutrally stratified background state defined by its uniform
+potential temperature. We solve a flow in a `FlatOrientation` (Box) configuration - this is representative of a large-eddy simulation. Several versions of the problem setup
+may be found in literature, but the general idea is to examine the vertical ascent of a
+thermal `bubble` (we can interpret these as simple representation of convective updrafts).
+
+## Description of experiment
+1) Dry Rising Bubble (circular potential temperature perturbation)
+2) Boundaries
+   - `Impenetrable(FreeSlip())` - no momentum flux, no mass flux through walls.
+   - `Impermeable()` - non-porous walls, i.e. no diffusive fluxes through walls.
+   - Laterally periodic
+3) Domain - 2500m[horizontal] x 2500m[horizontal] x 2500m[vertical]
+4) Resolution - 50m effective resolution
+5) Total simulation time - `1000s`
+6) Mesh Aspect Ratio (Effective resolution) 1:1
+7) Overrides defaults for
+   - CPU Initialisation
+   - Time integrator
+   - Sources
+   - Smagorinsky Coefficient
+8) Default settings can be found in `src/Driver/<files>.jl`
+
+!!! note
+    This experiment setup assumes that you have installed CLIMA according to
+    the instructions on the landing page. We suggest that you attempt to run the
+    `runtests.jl` file in the main `CLIMA` directory to ensure that packages and
+    `git` histories are consistently installed / managed. We assume the users' familiarity
+    with the conservative form of the equations of motion for a compressible fluid (see the `AtmosModel` page).
+
+    The following topics are covered in this example
+    - Package requirements
+    - Defining a `model` subtype for the set of conservation equations
+    - Defining the initial conditions
+    - Applying boundary conditions
+    - Applying source terms
+    - Choosing a turbulence model
+    - Adding tracers to the model
+    - Choosing a time-integrator
+    - Choosing diagnostics (output) configurations
+
+    The following topics are not covered in this example
+    - Defining new boundary conditions
+    - Defining new turbulence models
+    - Building new time-integrators
+    - Adding diagnostic variables (beyond a standard pre-defined list of variables)
+
+
+```math
+\tau = - 2 \nu \mathrm{S}
+```
+[`ConstantViscosityWithDivergence`](@ref constant-viscosity)\
+### [Pricipal Invariants](@id tensor-invariants)
+
+
+## Boilerplate (Using Modules)
+Before setting up our experiment, we recognize that we need to import
```

Comment:
We should consider have a generic introduction to Julia that tutorials can refer to.